src/teleop/gui/src/components/ArmControls.vue

```diff
@@ -49,13 +49,17 @@
         @change="toggleArmLaser()"
       />
     </div>
+    <div>
+      <CalibrationCheckbox :name="'Joint B Calibration'" :joint_name="'joint_b'" :calibrate_topic="'ra_is_calibrated'"/>
```

Comment:
each attribute is specified by "" and each one takes in a string, so that's why there's both "" and ''.

Reply:
Not sure if this would work but you could maybe do
```suggestion
      <CalibrationCheckbox name="Joint B Calibration" joint_name="joint_b" calibrate_topic="ra_is_calibrated"/>
```